mikupad.html

```diff
@@ -2031,6 +2195,8 @@
 						disabled=${!!cancel} value=${openaiPresets} onValueChange=${setOpenaiPresets}/>`}
 				<${InputBox} label="Seed (-1 = random)" type="text" inputmode="numeric"
 					readOnly=${!!cancel} value=${seed} onValueChange=${setSeed}/>
+				<${InputBox} label="Max Context Length (estimate, characters * 3.3)" type="text" inputmode="numeric"

```

Comment:
I feel like it's unnecessary to mention that this will be used as an estimate.
```suggestion
				<${InputBox} label="Max Context Length" type="text" inputmode="numeric"

```